spec/basket_spec.rb

```diff
@@ -2,19 +2,35 @@
 
 RSpec.describe Basket do
 
-  it "creates basket" do
-    expect Basket.new.is_a? Basket
+  let(:product1) { Product.new(name: "Football", price: 79) }
+  let(:product2) { Product.new(name: "Basketball", price: 60) }
+  let(:basket1) { Basket.new([product1, product2]) }
```

Comment:
do you have `basket2` somewhere? I would use `basket`, `foot_ball` and `basket_ball` instead.